screens/booking/MyBooking.tsx

```diff
@@ -1,19 +1,66 @@
 import { NativeStackScreenProps } from "@react-navigation/native-stack";
-import { View, Text } from "react-native";
-import { SafeAreaView } from "react-native-safe-area-context";
+import React from "react";
+import { View, StyleSheet, Pressable, SafeAreaView } from "react-native";
 
+import ActiveTimeSlot from "../../components/booking/ActiveTimeSlot";
+import CompletedTimeSlot from "../../components/booking/CompletedTimeSlot";
+import SubHeaderText from "../../components/text/SubHeaderText";
+import Colors from "../../constants/color";
 import { RootParamList } from "../../types";
 
+
 export type MyBookingProps = NativeStackScreenProps<RootParamList, "MyBooking">;
 
-const MyBooking: React.FC<MyBookingProps> = () => {
+const MyBooking: React.FC<MyBookingProps> = ({ navigation }) => {
+  const onPress = () => {
+    navigation.navigate("AllBookings");
+  };
   return (
-    <View>
+    <View style={styles.container}>
       <SafeAreaView>
-        <Text>My booking</Text>
+        <SubHeaderText text="Active Session" textStyle={styles.activeSession} />
+        <ActiveTimeSlot licensePlate="กบ 1454" space="space h3" timeRemaining="01hr:30min" />
+        <View style={styles.completedSessionsWrapper}>
+          <SubHeaderText
+            text="Completed Sessions"
+            textStyle={styles.completedSessionsText}
+          />
+          <Pressable onPress={onPress}>
+            <SubHeaderText text="View all" textStyle={styles.viewAllText} />
+          </Pressable>
+        </View>
+        <CompletedTimeSlot
+          licensePlate="หส 4142"
+          space="Spce f2"
+          date="13/03/41"
+          time="01:42 am"
```

Comment:
You should provide a util that format the data into those formats since the raw data won't come in this form.

Reply:
Done.